examples/editor/src/PluginsConfig.js

```diff
@@ -7,6 +7,7 @@ import { LINK_TYPE } from 'wix-rich-content-plugin-link';
 import { VIDEO_TYPE } from 'wix-rich-content-plugin-video';
 import { GIPHY_TYPE } from 'wix-rich-content-plugin-giphy';
 import { MAP_TYPE } from 'wix-rich-content-plugin-map';
+import {BUTTON_TYPE} from 'wix-rich-content-plugin-button';
```

Comment:
this will fail the linter, make sure you have eslint and prettier properly configured
```suggestion
import { BUTTON_TYPE } from 'wix-rich-content-plugin-button';
```